### ISSUE TITLE
EuroClojure 2017 Slide 10 Possible Error

### ISSUE THREAD

**Issue by pwhittin** [suprematic/otplike#20]
On slide 10 of the EuroClojure 2017 presentation the following code is displayed:

```clojure
(proc-defn process2 []
  (receive!
    [:message m]
    (println message))
  (recur))

(proc-defn process1 []
  (let [p2 (process/spawn-link process2)]
    (receive 
      m
      (! p2 m))
    (recur)))

(let [p1 (process/spawn process1)]
  (! p1 :unsupported-message))
```

In "process1" is the line "(receive" supposed to be "(receive!"?

**Comment by pwhittin** [comment]
Furthermore, it looks like the line "(println message))" should be "(println m))"

**Comment by sskorokhodov** [comment]
You are right. I'll fix this ASAP. Thank you very much for the feedback!

**Comment by pwhittin** [comment]
My pleasure!  Do you remember that we spoke and had dinner a couple of times during EuroClojure 2017, and last year at EuroClojure 2016?

**Comment by sskorokhodov** [comment]
Yeah, sure! Nice to meet you online :)

**Comment by sskorokhodov** [comment]
Fixed by commit 77dfcd2 of suprematic.github.io project.
